src/Symfony/Component/Notifier/Bridge/GatewayAPI/GatewayAPITransport.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\GatewayAPI;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Piergiuseppe Longo <piergiuseppe.longo@gmail.com>
+ * @experimental in 5.2
+ */
+final class GatewayAPITransport extends AbstractTransport
+{
+    protected const HOST = 'gatewayapi.com';
```

Comment:
Why protected?

Reply:
I don't know. It is protected in other Transports, like: FreeMobileTransport, SmsapiTransport, NexmoTransport, etc.

Reply:
Let's make it private then =)

Reply:
I don't agree with you. The host is protected in the other Transports and `AbstractTransport`, so it would be better to keep it protected to be consistent.